src/voltorb_flip/voltorb_flip_input.c

```diff
@@ -251,82 +335,91 @@ static int ov122_021E9020(Ov122_021E8CFC *a0) {
 }
 
-static void ov122_021E9154(Ov122_021E8CFC *a0, int newFocus) {
+static void VoltorbFlipInputHandler_UpdateBoardCursorAnim(VoltorbFlipInputHandler *inputHandler, int newFocus) {
     int indicatorType;
 
-    if (newFocus < 25) {
-        if (a0->memoFocused) {
+    if (newFocus < VOLTORB_FLIP_INPUT_GRID_NUM) { // In grid
+        if (inputHandler->memoFocused) {
             indicatorType = 24; // Disabled pencil
-        } else if (a0->memoOpen) {
+        } else if (inputHandler->memoOpen) {
             indicatorType = 23; // Pencil
         } else {
             indicatorType = 18; // Card focus
         }
-    } else if (newFocus == 25) {
-        indicatorType = 7; // Open/close Memo button
-    } else if (newFocus == 26) {
-        indicatorType = 4; // Quit button
+    } else if (newFocus == VOLTORB_FLIP_INPUT_MEMO) { // Memo
+        indicatorType = 7;                            // Open/close Memo button
+    } else if (newFocus == VOLTORB_FLIP_INPUT_QUIT) { // Quit
+        indicatorType = 4;                            // Quit button
     } else {
         GF_ASSERT(FALSE);
     }
 
-    if (a0->memoOpen) {
-        ManagedSprite_SetPaletteOverride(a0->unk0, 4);
+    if (inputHandler->memoOpen) {
+        ManagedSprite_SetPaletteOverride(inputHandler->boardCursorSprite, 4);
     } else {
-        ManagedSprite_SetPaletteOverride(a0->unk0, 2);
+        ManagedSprite_SetPaletteOverride(inputHandler->boardCursorSprite, 2);
     }
-    ManagedSprite_SetAnimNoRestart(a0->unk0, indicatorType);
+    ManagedSprite_SetAnimNoRestart(inputHandler->boardCursorSprite, indicatorType);
 }
 
-static void ov122_021E91AC(Ov122_021E8CFC *a0, u32 newFocus, int a2, int a3) {
-    if ((a3 == 0 || newFocus != 25) && newFocus != a2 && newFocus < 28) {
+static void VoltorbFlipInputHandler_HandleChangeFocus(VoltorbFlipInputHandler *inputHandler, int newFocus, int oldFocus, MenuInputState inputMode) {
+    if ((inputMode == MENU_INPUT_STATE_BUTTONS || newFocus != VOLTORB_FLIP_INPUT_MEMO) && newFocus != oldFocus && newFocus < NELEMS(sTouchscreenHitboxes)) {
         // Updates visual focus indicator
-        ov122_021E9108(a0, newFocus);
+        VoltorbFlipInputHandler_SetFocus(inputHandler, newFocus);
         // Plays when you switch focus
         PlaySE(SEQ_SE_DP_SELECT);
     }
 }
 
-void ov122_021E91D0(Ov122_021E8CFC *a0, int a1, int a2) {
-    sub_02019F7C(a0->unk8, (u8)a1);
-    ov122_021E9108(a0, a1);
+// UnkStruct_02019BA4_unk18_func callbacks
+
+static void VoltorbFlipInputCB_OnButtonPress(void *data, int new, int prev) {
+    VoltorbFlipInputHandler *inputHandler = data;
+
+    GridInputHandler_SetNextInput(inputHandler->gridInputHandler, (u8)new);
+    VoltorbFlipInputHandler_SetFocus(inputHandler, new);
     PlaySE(SEQ_SE_DP_SELECT);
 }
 
-void ov122_021E91F4(Ov122_021E8CFC *a0, int a1, int a2) {
+static void VoltorbFlipInputCB_OnTouchSwitch(void *data, int new, int prev) {
```

Comment:
QoL: Rename to indicate no-op?

Reply:
imo better to name based on what it's passed to in this case